packages/atomic/src/locales.json

```
@@ -6535,6 +6535,36 @@
     "zh-CN": "哎呀！在生成答案时出错了。",
     "zh-TW": "糟糕！在生成答案時出了些問題。"
   },
+  "generated-answer-feedback-header": {
```

These keys should be about the text content as much as possible, and not necessarily where or why they are used.

ie: 

```suggestion
  "feedback": {
```

I know there are some other keys in this file which do not follow this rule, but 🤷